extensions/wikia/Email/Controller/FounderController.class.php

```diff
@@ -209,3 +210,114 @@ protected function getFooterEncouragement() {
 			->parse();
 	}
 }
+
+class FounderActiveController extends FounderController {
+
+	/**
+	 * Define this and do nothing since we don't need the checks of our parent
+	 */
+	public function initEmail() {
+		// NOOP
+	}
+
+	public function getSubject() {
+		return $this->getMessage( 'emailext-founder-active-subject' )->text();
+	}
+
+	/**
+	 * @template multiAvatarLayout
+	 */
+	public function body() {
+		$this->response->setData( [
+			'salutation' => $this->getSalutation(),
+			'summary' => $this->getSummary(),
+			'avatarAndDetailsList' => $this->getChangeList(),
+			'buttonText' => $this->getChangesLabel(),
+			'buttonLink' => $this->getChangesLink(),
+			'contentFooterMessages' => [
+				$this->getMessage( 'emailext-founder-active-footer-1' )->text(),
+				$this->getMessage( 'emailext-founder-active-footer-2' )->text(),
+			],
+			'hasContentFooterMessages' => true
+		] );
+	}
+
+	protected function getSummary() {
+		return $this->getMessage( 'emailext-founder-active-summary' )->parse();
+	}
+
+	private function getChangeList() {
+		$changes = $this->getRecentActivity();
+		$changeList = [];
+
+		foreach ( $changes as $event ) {
+			try {
+				$eventUser = $this->getUserFromName( $event[ 'user' ] );
+			} catch ( ControllerException $e ) {
+				continue;
```

Comment:
If we can't find a user from their name, don't sweat it, just skip this user and move on to the next.